@types/frame/state.d.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
I think we can get rid of this exception and just move the `Colorway` enum out of this file